kube/destroy-info-schema.yaml

```diff
@@ -12,7 +12,8 @@ spec:
     spec:
       containers:
       - name: toolbox
-        image: quay.io/ukhomeofficedigital/hocs-toolbox:branch-main # TODO: make this :latest
+        image: quay.io/ukhomeofficedigital/hocs-toolbox
+        imagePullPolicy: Always
```

Comment:
This is implicit

Reply:
This is not true, it sets to 'IfNotPresent' by default: https://kubernetes.io/docs/concepts/containers/images/#updating-images

This has caused us confusion in the past whereby a new latest is not pulled as it believes it still has the latest.